src/languages/koCSSLanguage.py

```diff
@@ -52,6 +53,14 @@ class koCSSCommonLanguage(KoLanguageBase):
     searchURL = "http://www.google.com/search?q=site%3Ahttp%3A%2F%2Fwww.w3schools.com%2Fcss+%W"
 
     sample = """
+
```

Comment:
I thought it would be better to mention all most-used CSS keywords/properties/etc. in the sample to make the scheme CSS syntax modifying more understandable